orttraining/orttraining/test/python/launch_test.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import sys
+import argparse
+import subprocess
+
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
+
+log = logging.getLogger("Build")
+
+def run_subprocess(args, cwd=None, env={}):
+    log.info("Running subprocess in '{0}'\n{1}".format(cwd or os.getcwd(), args))
+    my_env = os.environ.copy()
+    my_env.update(env)
+    completed_process = subprocess.run(args, cwd=cwd, check=True, env=my_env)
+    log.debug("Subprocess completed. Return code=" +
+              str(completed_process.returncode))
+    return completed_process
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--process_args", required=True, help="process args.")
+    parser.add_argument("--cwd", help="cwd")
+    parser.add_argument("--dll_path", help="dll path.")
+    parser.add_argument("--env", help="env variables.")
```

Comment:
how about:
```
parser.add_argument("--env", help="...", nargs=2, action="append", default=[])
```
to get a list of 2 element lists? can let argparse do the validation